exchange.lisp

```diff
@@ -375,7 +372,7 @@ need-to-use basis, rather than upon initial loading of the exchange API.")
 
 (defmethod initialize-instance :after ((tracker trades-tracker) &key)
   (adopt tracker (setf (slot-value tracker 'fetcher)
-                       (make-instance 'trades-fetcher :delegates `(,tracker)))))
+                       (make-instance 'trades-fetcher :boss tracker))))
```

Comment:
the naming "boss" is shorter, and might be more intuitive in the context of actors influencing and monitoring the behavior of other actors.